src/main/java/com/actiontech/dble/backend/datasource/PhysicalDBPool.java

```diff
@@ -381,7 +381,7 @@ private boolean initSource(int index, PhysicalDatasource ds) {
             if (ds.getTotalConCount() <= 0) {
                 ds.initMinConnection(null, true, getConHandler, null);
             } else {
-                LOGGER.info("connection with null schema do not create,because testConnection in pool");
```

Comment:
can you also fix `because ...`?

Reply:
No,I think testConnection is the real cause that created the connection with null schema.

Reply:
> No,I think testConnection is the real cause that created the connection with null schema.

I mean the English grammar... not the reason...

Reply:
Okey, could ‘because we tested the connection of the datasource at first’ be appropriate?

Reply:
> Okey, could ‘because we tested the connection of the datasource at first’ be appropriate?

:100: